snare.py

```diff
@@ -233,6 +234,12 @@ def handle_request(self, request, payload):
                     self.writer, status=404, http_version=request.version
                 )
         response.add_header('Server', self.run_args.server_header)
+        if 'cookies' in data and 'sess_uuid' in data['cookies']:
+            previous_sess_uuid = data['cookies']['sess_uuid']
+        else:
+            previous_sess_uuid = None
+        if previous_sess_uuid is None or not previous_sess_uuid.strip():
+            response.add_header('Set-Cookie', 'sess_uuid='+event_result['response']['message']['sess_uuid'])
```

Comment:
Snare can updates automatically, but tanner can't. If I merge these changes, this line can broke everything, because old tanner don't return `sess_uuid` in `message`. Could you please check this value before string concatenation?